backend/app/views/spree/admin/reimbursements/edit.html.erb

```diff
@@ -16,12 +16,12 @@
     <table class="index reimbursement-return-items">
       <thead>
         <tr>
-          <th><%= Spree.t(:product) %></th>
-          <th><%= Spree.t(:preferred_reimbursement_type) %></th>
-          <th><%= Spree.t(:reimbursement_type_override) %></th>
-          <th><%= Spree.t(:pre_tax_refund_amount) %></th>
-          <th><%= Spree.t(:total) %></th>
-          <th><%= Spree.t(:exchange_for) %></th>
+          <th><%= Spree::Product.model_name.human %></th>
+          <th><%= Spree::ReturnItem.human_attribute_name(:preferred_reimbursement_type_id) %></th>
+          <th><%= Spree::ReturnItem.human_attribute_name(:override_reimbursement_type_id) %></th>
+          <th><%= Spree::ReturnItem.human_attribute_name(:amount) %></th>
+          <th><%= Spree::Reimbursement.human_attribute_name(:total) %></th>
```

Comment:
`Spree::ReturnItem` doesn't have a `total` column in the DB so I went with reimbursement to maintain the wording in my efforts to not add attributenames when there is no column.  In hindsight that was overly stringent.